ui/page/main_page.go

```diff
@@ -313,10 +357,19 @@ func (mp *MainPage) UnlockWalletForSyncing(wal *dcrlibwallet.Wallet) {
 // displayed.
 // Part of the load.Page interface.
 func (mp *MainPage) HandleUserInteractions() {
+	if mp.IsCurrencySettingUpdated {
+		mp.updateExchangeSetting()
+		mp.IsCurrencySettingUpdated = false
+	}
+
 	if mp.currentPage != nil {
 		mp.currentPage.HandleUserInteractions()
 	}
 
+	if mp.refreshValue.Clicked() && mp.usdExchangeSet {
```

Comment:
Would there even be a refresh button to click if the exchange rate setting isn't configured?
```suggestion
	if mp.refreshValue.Clicked() {
```